File: swift/etl/Sources/Etl/Etl.swift
//Solution goes in Sources
struct ETL {

    static func transform(_ array: [Int: [String]]) -> [String: Int] {
       var result = [String:Int]()       

       let _ = array.map { (key, value) in 
            value.map { 
                result[$0.lowercased()] = key
            }
       }
       return result
    }
}

File: go/etl/etl.go
package etl

import "strings"

func Transform(input map[int][]string) map[string]int {
	result := make(map[string]int)

	for k, v := range input {

		for _, s := range v {
			lowercaseString := strings.ToLower(s)

			result[lowercaseString] = k
		}
	}

	return result
}
